src/components/Videos.vue

```diff
@@ -40,7 +41,8 @@
 				preload="metadata"
 				@ended="donePlaying"
 				@canplay="doneLoading"
-				@loadedmetadata="onLoadedMetadata">
+				@loadedmetadata="onLoadedMetadata"
+				v-html="tracks">
```

Comment:
```suggestion
				>
				<track v-for="track in tracks"  
				            :src="track.src" 
				            :label="track.label"
				            kind="captions"
				            :srclang="track.lang" />
```

You should use `v-for` to handle the tracks instead of injecting HTML, this will also remove GitHub's XSS warnings. For this to work `tracks` must return an array of objects with src, label and lang keys. I wrote this code from what I think it should look like, but it will likely not work due to some typo :stuck_out_tongue: